README.md

```diff
@@ -190,3 +190,22 @@ The final summary is saved to the graph state as well:
 There are [various ways](https://langchain-ai.github.io/langgraph/concepts/#deployment-options) to deploy this graph.
 
 See [Module 6](https://github.com/langchain-ai/langchain-academy/tree/main/module-6) of LangChain Academy for a detailed walkthrough of deployment options with LangGraph.
+
+## Running as a Docker container
+
+The included `Dockerfile` only runs LangChain Studio with ollama-deep-researcher as a service, but does not include Ollama as a dependant service. You must run Ollama separately and configure the OLLAMA_BASE_URL environment variable.
+
+Clone the repo and build an image:
+```
+$ docker build -t ollama-deep-researcher .
+```
+
+Run the container:
+```
+$ docker run --rm -it -p 2024:2024 \
+  -e TAVILY_API_KEY="tvly-***YOUR_KEY_HERE***" \
+  -e OLLAMA_BASE_URL="http://host.docker.internal:11434/" \
+  ollama-deep-researcher
+```
+
+Then visit link: `https://smith.langchain.com/studio/thread?baseUrl=http://127.0.0.1:2024`
```

Comment:
Might it be worth making some extra comment that the http://0.0.0.0:2024 presented in the logs cannot be used, (just for extra clarity)?

Reply:
Good idea, the log message may confuse some. Will do